flink/v1.18/build.gradle

```diff
@@ -63,6 +63,7 @@ project(":iceberg-flink:iceberg-flink-${flinkMajorVersion}") {
       // These artifacts are shaded and included in the orc-core fat jar
       exclude group: 'com.google.protobuf', module: 'protobuf-java'
       exclude group: 'org.apache.hive', module: 'hive-storage-api'
+      exclude group: 'org.slf4j'
```

Comment:
out of curiosity: why was this needed?

Reply:
We specify 2.x slf4j as an Iceberg dependency. But these dependencies carry a 1.x slf4j transitive dependency. When both 1.x and 2.x slf4j is on classpath the the loggers are not working, so it is very hard to follow what happens during the tests